tests/ui/issue-13560.rs

```diff
@@ -1,4 +1,5 @@
 // run-pass
+// ignore-wasm32-unknown-unknown (apparently the compiler still has the original bug there??)
```

Comment:
i wonder if we should file a bug for this..?

Reply:
Good idea, done: https://github.com/rust-lang/rust/issues/110172